shared/src/business/entities/cases/Case.addDocketEntry.test.js

```diff
@@ -4,7 +4,25 @@ const {
 const { Case } = require('./Case');
 
 describe('addDocketEntry', () => {
-  it('attaches the docket entry to the case', () => {
+  it('should throw when docket entry is a STIN', () => {
```

Comment:
I'm now wondering if these titles should be different.
```suggestion
  it('should throw when docket entry to be added to the docket record is a STIN', () => {
```

Reply:
I feel like that may not be necessary as the method under test is conveying the adding of the docket entry!